File: Basic of Python/power series.py
x=int(input("Enter the value of x:"))
n=int(input("Enter the value of n:"))
sum=0
for i in range(0,n):
    a=x**(2*i)
    if (2*i)==0:
        b=1
    else:
        b=2*i
    sum+=(a/b)
print("Enter the sum of the power series for x =",x,"and n =",n,"is",sum)

File: Basics of C Programming/power_series.c
#include <stdio.h>
#include <math.h>

//1 + x^2/2 + x^4/4 + ...... + x^n/n.

int main(){
    double x,n,d,b,sum=0;
    printf("Enter the value of x:");
    scanf("%lf",&x);
    printf("Enter the value of n:");
    scanf("%lf",&n);
    for (int i=0;i<n;i++){
        b=pow(x,2*i);
        d=2*i;
        if (d==0){
            d=1;
        }
        double c=b/d;
        sum+=c;
    }
    printf("The sum of the power series is %0.3lf",sum);
    return 0;
}